docs/dev_guide/contents/extending_fido.rst

```diff
@@ -82,6 +78,22 @@ Since `~sunpy.net.scraper.Scraper` doesn't currently support generating director
 The search method should in this case, generate a ``baseurl`` dependant on the values of these attrs, and then call ``super().search`` or `~sunpy.net.scraper.Scraper` for each ``baseurl`` generated.
 For an example of a complex modification of the ``search()`` method see the implementation of `.SUVIClient.search`.
 
+Customizing the Downloader
+--------------------------
+
+There is no method for a client creator to override the `parfive.Downloader` that is used to fetch the final files.
+This is because we allow a user to setup a custom `parfive.Downloader` by passing it to the ``downloader`` argument of :meth:`Fido.fetch`.
```

Comment:
While technically true, the more important statement is that all downloads from all clients share a single `Downloader` instance, be that the one passed by the user or the one created in `Fido.fetch`.

```suggestion
This is because all downloads made by a single call to :meth:`Fido.fetch` share one instance of `parfive.Downloader`.
```